docs/source/setup.rst

```diff
@@ -0,0 +1,534 @@
+      --database.password "${GRAPHDB_ROOT_PASSWORD:-changeme}"
+
+Prepare graph database for ``ckcore``
+-------------------------------------
+
+Now you need to prepare the graph database for ``ckcore``.
+
+You will add an user for ``ckcore``, create a database and grant privileges for the user to access the database.
+
+Our defaults run fine but things like ``GRAPHDB_ROOT_PASSWORD`` or ``CKCORE_GRAPHDB_PASSWORD`` need to be changed for non-laptop-environments.
+
+.. code-block:: bash
+    :caption: Run ArangoSH to configure graph database
+
+    $ arangosh --console.history false --server.password "${GRAPHDB_ROOT_PASSWORD:-changeme}"
+    > const users = require('@arangodb/users');
+    > users.save('${CKCORE_GRAPHDB_LOGIN:-cloudkeeper}', '${CKCORE_GRAPHDB_PASSWORD:-changeme}');
+    > db._createDatabase('${CKCORE_GRAPHDB_DATABASE:-cloudkeeper}');
+    > users.grantDatabase('${CKCORE_GRAPHDB_LOGIN:-cloudkeeper}', '${CKCORE_GRAPHDB_DATABASE:-cloudkeeper}', 'rw');
+
+.. _prometheus:
+
+Prometheus
+==========
+
+Run
+---
+
+In this example we expect a configuration at ``/usr/local/tsdb/prometheus.yml`` with this configuration
+
+.. code-block:: yaml
+    :caption: /usr/local/tsdb/prometheus.yml configuration.
+
+    global:
+        scrape_interval: 120s 
+        evaluation_interval: 120s
+
+        alerting:
+        alertmanagers:
+            - static_configs:
+                - targets:
+                # - alertmanager:9093
+
+        rule_files:
+        # - "first_rules.yml"
+        # - "second_rules.yml"
+
+        scrape_configs:
+        - job_name: "prometheus"
+            static_configs:
+            - targets: ["localhost:9090"]
+
+        - job_name: "ckmetrics"
+            static_configs:
+            - targets: ["localhost:9955"]
+
+
+.. code-block:: bash
+    :caption: Create data directory and run Prometheus
+
+    $ mkdir -p ${TSDB_DATABASE_DIRECTORY:-/data/tsdb}
+    $ /usr/local/tsdb/prometheus --config.file=${TSDB_CONFIG_FILE:-/usr/local/tsdb/prometheus.yml} \
+      --storage.tsdb.path=${TSDB_DATABASE_DIRECTORY:-/data/tsdb} \
+      --storage.tsdb.retention.time=${TSDB_RETENTION_TIME:-730d} \
+      --web.console.libraries=/usr/local/tsdb/console_libraries \
+      --web.console.templates=/usr/local/tsdb/consoles \
+      --web.enable-lifecycle \
+      --web.enable-admin-api
+
+Install Cloudkeeper components
+******************************
+
+.. _setup-ckcore:
+
+ckcore
+======
+
+Install ckcore
+--------------
+
+You install ckcore via python pip directly from our git repository.
+Please make sure you have git installed.
+First you need to install :ref:`cklib` as a dependency to :ref:`setup-ckcore`.
+
+.. code-block:: bash
+    :caption: Install cklib und ckcore
+
+    $ pip3 install "git+https://github.com/someengineering/cloudkeeper.git@main#egg=cklib&subdirectory=cklib"
+    $ pip3 install "git+https://github.com/someengineering/cloudkeeper.git@main#egg=ckcore&subdirectory=ckcore"
+
+Usage
```

Comment:
Lets not repeat the large Usage block from the README in here. Tell the user that `--help` will show a list of all cli args and then go straight into the ones we'd use for the setup.

Reply:
Done via 9cef348